api/REST-API.md

```diff
@@ -1,191 +1,574 @@
+  "last_name": "Brabon",
+  "email": "lbrabon4@umich.edu",
+  "gender": "Female",
+  "ip_address": "47.137.187.14"
+  }]
+
+let invoices = [
+  {
+  "id": 6,
+  "product": "Island Oasis - Ice Cream Mix",
+  "price": "$0.78",
+  "currency": "BRL",
+  "quantity": 17,
+  "type": "Restaurants"
+  }, 
+  {
+  "id": 7,
+  "product": "Ice Cream - Turtles Stick Bar",
+  "price": "$9.23",
+  "currency": "EUR",
+  "quantity": 9,
+  "type": "Marine Transportation"
+  }, 
+  {
+  "id": 8,
+  "product": "Shrimp - Black Tiger 8 - 12",
+  "price": "$2.50",
+  "currency": "COP",
+  "quantity": 20,
+  "type": "Computer Communications Equipment"
+  }, 
+  {
+  "id": 9,
+  "product": "Wine - Blue Nun Qualitatswein",
+  "price": "$8.52",
+  "currency": "IDR",
+  "quantity": 64,
+  "type": "Packaged Foods"
+  }, 
+  {
+  "id": 10,
+  "product": "Truffle Shells - White Chocolate",
+  "price": "$7.51",
+  "currency": "EUR",
+  "quantity": 65,
+  "type": "Industrial Specialties"
+  }]
 
-{
-   "actionName": "getEvent",
-   "parameters": {
-      { "eventId": 123 }
-   }
-}
 ```
 
-The response to the above might be:
+Once this is done, we will create our first route. Remember our planning? This is the route we want to create:
 
-```
-HTTP/1.1 200 OK
+|Route|HTTP Method|
+|-|-|
+|`/customers`|`GET`|
 
-{
-   "result": "OK",
-   "errors": [],
-   "data": {
-      "eventId": 123,
-      "eventName": "Corgi Con"
-   }
-}
+With Express, routing is as easy as chaining the HTTP method to the `route()` method, taking care of putting the route you want to lookup inside of the `route()` method parenthesis.
+
+```javascript
+app
+  .route('/customers')
+  .get(
+    (req, res) => {
+      res.status(200).send(customers);
+    }
+  )
 ```
 
-Or if it couldn't find one:
+Now, if you test your route with Postman or by opening a browser and going to `http://localhost:3000/customers`, you should see:
 
+```json
+[{"id":1,"first_name":"Fremont","last_name":"Broader","email":"fbroader0@bloglines.com","gender":"Male","ip_address":"23.27.246.1"},{"id":2,"first_name":"Georgetta","last_name":"Blamey","email":"gblamey1@wisc.edu","gender":"Female","ip_address":"17.110.6.159"},{"id":3,"first_name":"Meghann","last_name":"Quillinane","email":"mquillinane2@surveymonkey.com","gender":"Female","ip_address":"237.88.226.148"},{"id":4,"first_name":"Kerby","last_name":"Mate","email":"kmate3@si.edu","gender":"Male","ip_address":"81.44.87.187"},{"id":5,"first_name":"Loren","last_name":"Brabon","email":"lbrabon4@umich.edu","gender":"Female","ip_address":"47.137.187.14"}]
 ```
-HTTP/1.1 200 OK
 
-{
-   "result": "FAIL",
-   "errors": ["Unable to find event with ID 123"],
-   "data": null
-}
+So, the same way, you can setup the second GET method, as planned:
+
+|Route|HTTP Method|
+|-|-|
+|`/invoices`|`GET`|
+
+With Express, routing is as easy as chaining the HTTP method to the `route()` method, taking care of putting the route you want to lookup inside of the `route()` method parenthesis.
```

Comment:
```suggestion
```
Was this repeated intentionally?

Reply:
Not at all... Nice catch!

Fixed in [e76944c](https://github.com/Techtonica/curriculum/pull/1410/commits/e76944c2ebe4ae93d9adf1edab007af9e6589c7b)